### DISCUSSION TITLE
Automatic acceptation of the xtts_v2 model license

### DISCUSSION THREAD

**Discussion by n0vember** [idiap/coqui-ai-TTS#78] [Q&A] [answered]
Hi!

I am using coqui-tts inside a container and each time I run a new container I get the question to accept or not the license for the model usage. Is there a way to accept it in an automated way by placing a file in the container or by setting on option in the model loading?

**Comment by eginhard** [comment, marked as answer]
You can set the `COQUI_TOS_AGREED` environment variable to 1.

**Comment by n0vember** [comment]
Exactly the information I was looking for. Thank you.
